public/js/controllers/traffic_graphs_controller.js

```diff
@@ -298,7 +334,14 @@ export class TrafficGraphsController {
             .shapeWidth(12)
             .orient("vertical")
             .scale(colorSent)
-            .labels(operators);
+            .labels(operators)
+            .on("cellover", function () {
```

Comment:
```suggestion
            .on("cellover", function () {
```
I think we should refer to this as legends and not cells so on `.on("legendhover", function () {`

Reply:
We're using d3-legend. There are three custom event types we can bind to the legend: "cellover", "cellout", and "cellclick".